src/backend/simple.rs

```diff
@@ -75,6 +75,11 @@ where
         symbol
     }
 
+    #[inline]
```

Comment:
I am not convinced that `#[inline]` is needed for APIs that are not expected to be called very often.